TRUCSBot/Commands/GameNightSuggestionCommands.cs

```diff
@@ -88,5 +101,51 @@ public async Task AddToBoard(CommandContext ctx, [Description("The title of the
                 _logger.LogError("Error occurred posting game suggestion message", ex);
             }
         }
+
+        // https://www.csharpstar.com/csharp-string-distance-algorithm/
```

Comment:
Can we change this to a documentation comment and also add a short explanation of what it does?